Changes

```diff
@@ -1,3 +1,8 @@
+1.27 2021-04-28
+  - removed ill advised restriction on Event::Inline, used to
+    integrate 'with' Inline::* modules
+  - *no* version bump
```

Comment:
Just a single line with the change would have been fine. I avoid commentary about other people's code like "ill-advised" as well. You don't need to put a version header here, that's a release-manager job.

Reply:
will fix